backend/app/controllers/books_controller.rb

```diff
@@ -23,11 +25,11 @@ def create
   private
 
   def book_params
-    params.require(:book).permit(:title, :description)
+    params.require(:book).permit(:title, :description, :image)
```

Comment:
This is more of a `nit`, but keep this in mind when you start working on production-level projects.

Sometimes the experience is better if you:
- directly upload the image from the frontend, then just pass a URL in the backend.
- have a dedicated "image" endpoint for uploading or deleting images. Then you can ping this endpoint as the user is picking images they want to upload
- Bundling the `image` here would mean that, if there is a validation error, the uploaded image is scrapped. This might irritate your user if they waited a long time for the upload to happen to be hit by an error message